macros/edr/dbt_artifacts/upload_source_freshness_results.sql

```diff
@@ -0,0 +1,28 @@
+{% macro upload_source_freshness_results() %}
+  {% set source_freshness_results_relation = ref('elementary', 'source_freshness_results') %}
+  {% set sources_json_path = ref.config.target_path ~ '/sources.json' %}
```

Comment:
Just to be on the safe side - what do you think about validating the target_path is not null and only then doing this concatenation?

Reply:
I don't understand why check a condition that cannot occur.
How would `target_path` be `null`?
Even if you remove `target-path` from `dbt_project.yml`, it still defaults to `target`.